calculator.js

```diff
@@ -0,0 +1,142 @@
+let state = {
+  currentOperation: null, // это может быть сложение, умножение, вычитание, etc. Ключи: -, + и тд
+  immediateResult: 0, // "мгновенный" результат. Используется только для рендеринга, !не для расчетов!
+  fixedResult: 0, // используется как опорное значение. На его основе считается immediateResult
+  currentOperand: "", // это буфер, хранящий текущее число. Чтоб не плюсовали 1, потом 2, потом 3, а сразу 123
+};
+
+const setState = (key, payload) => {
+  state = { ...state, [key]: payload };
+  renderImmediateResult();
+  console.log(state);
+};
+
+function setInitialState() {
+  setState("currentOperation", null);
+  setState("immediateResult", 0);
+  setState("fixedResult", 0);
+  setState("currentOperand", "");
+}
+
+const DIGIT_BUTTONS = document.querySelectorAll(".digit");
+const OPERATOR_BUTTONS = document.querySelectorAll(".operation");
+const BTN_VALUES = Array.from(document.querySelectorAll(".button")).map(
+  (el) => el.innerHTML
+);
+const KEYBOARD = document.querySelector(".keyboard");
+const INPUT = document.querySelector("#myInput");
```

Comment:
do we really need ID for this element? If possible, let's replace with class selector

Reply:
done

Reply:
@artur-barmin done